translations/strings.yaml

```diff
@@ -90,6 +90,15 @@ serversView:
     comment: Label for the “recommended” servers tab
   searchPlaceholder: Search countries
   searchNoResultsLabel: No results. Try a different search
+  recommendedRefreshlLoadingLabel:
+    value: Checking... %1%
+    comment: Loading string while recommended servers are being calculated. Argument is the percentage of completion as an integer (i.e. 25)
+  recommendedRefreshLastUpdatedLabel:
+    value: Last updated %1.
+    comment: The time (or in extremely rare circumstances, the date, day of the week, or "yesterday") which the recommended servers list was last updated. Argument is the time or date/day
```

Comment:
How is the date formatted? Where is "yesterday" coming from?

Could we have a clearer split between sentence and date, i.e. `Last update(d): %1`?

Reply:
The date is formatted identically to our in-app messages. A quick summary: 

- Timestamps can be “\<time\> “Yesterday”, “\<day of the week\>”, or “\<date\>”
  - Examples: “9:41 AM”, “Yesterday”, “Monday”, “Friday”, “7/5/22”
  -"Yesterday" is translated [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/082194b604b0b67c8d657b9d8252dc9636eaeb81/translations/strings.yaml#L743) (though now I am wondering if the capitalization is okay - what do you think?)
  -[Days of the week](https://doc.qt.io/qt-6.2/qlocale.html#dayName) and date formatting are handled by Qt on a locale basis

> Could we have a clearer split between sentence and date, i.e. Last update(d): %1?

@chrisrooneys - thoughts?

Reply:
>         -"Yesterday" is translated [here](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/082194b604b0b67c8d657b9d8252dc9636eaeb81/translations/strings.yaml#L743) (though now I am wondering if the capitalization is okay - what do you think?)

It's not, that's why I asked ;-) 

You need a new string that will be used exclusively in this context.

Reply:
If I'm understanding, we need a separate string for "yesterday" as well as using a colon to divide the date from the rest of the sentence? I'm good with both of those. @MattLichtenstein let me know what you need.

Reply:
It's worth pointing out that the server latency data is refreshed on a 30 minute timer, so this string is almost always going to show "Today" except when the user has had their VPN turned on for more than a day (which blocks the refresh timer). It's not necessarily a problem, but it just makes this string a little less useful than you might expect.